Verification_Engine/Compute/TryGetValueFromSource.cs

```diff
@@ -247,6 +248,19 @@ private static Output<bool, object> TryGetValueFromSource(this object obj, strin
 
         /***************************************************/
 
+        private static PropertyInfo GetProperty(object obj, Type type, string sourceName)
+        {
+            var properties = type.GetProperties().Where(p => p.Name == sourceName).ToList();
```

Comment:
I do not think you need to cast `ToList` here, it is just an overhead (which may be costly in case of reflection being called many times)

Reply:
Fixed 👍